### PR TITLE
Feat/position pricing utils

### PR DESCRIPTION
**Pull request by sparqet**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Feature
- Testing

## What is the new behavior?

- implemented position pricing utils functions

## Does this introduce a breaking change?

No

### CODE REVIEW

File: src/pricing/position_pricing_utils.cairo [hunk 1]
```diff
@@ -168,18 +178,91 @@ struct PositionUiFees {
...
+        return price_impact_usd_for_virtual_inventory;
+    }
+
+    price_impact_usd
 }
 
 /// Called internally by get_price_impact_params().
-fn get_price_impact_usd_(
-    params: GetPriceImpactUsdParams,
+fn _get_price_impact_usd(
```
Comment (zarboq) [inline src/pricing/position_pricing_utils.cairo:215]: don't start with _

**Review by zarboq — COMMENTED** [review]
Some functions are missing documentation

**Review by zarboq — CHANGES_REQUESTED** [review]
Great PR overall :rocket:

Added some minor fix request

File: src/pricing/position_pricing_utils.cairo [hunk 2]
```diff
@@ -166,18 +177,91 @@ struct PositionUiFees {
...
-    // TODO
-    0
+    let initial_diff_usd = calc::diff(
+        open_interest_params.long_open_interest, open_interest_params.short_open_interest
+    );
+    let next_diff_usd = calc::diff(
+        open_interest_params.next_long_open_interest, open_interest_params.next_short_open_interest
+    );
+
+    let is_same_side_rebalance_first = open_interest_params
```
[the remark this answers is not in the document]
Comment (sparqet) [reply src/pricing/position_pricing_utils.cairo:226]: I tried to, but doesn't work

**Review by Sk8erboi84 — APPROVED** [review]

**Review by zarboq — APPROVED** [review]

**Pull request merged**